engine/SearchResult.js

```diff
@@ -0,0 +1,20 @@
+// When looking up content within the story (e.g. in Container.ContentAtPath),
```

Comment:
Would you mind removing these comments? I know I've done it myself, but in hindsight leaving comments from the original implementation is not great. To understand the code, reading the C# versions is preferable, and it drowns out comments that are specific to the js port.

Reply:
Yup, no problem.

You know what, I'll make another PR to add CONTRIBUTING.md detailing how to deal with this sort of thing. Comments, porting-style, etc.